app/build.gradle

```diff
@@ -113,6 +113,16 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.1'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
 
+    // Koin for Android
+    implementation "org.koin:koin-android:2.0.1"
+
+    // Koin Android Scope feature
+    implementation "org.koin:koin-android-scope:2.0.1"
+
+    // Koin Android ViewModel feature
+    implementation "org.koin:koin-android-viewmodel:2.0.1"
```

Comment:
@SergeiMikhailovskii I would recommend you to define a single version variable for your `koin` dependencies in your gradle file same as you did for `kotlin_version`

Reply:
@anton-novikau should I inject anything else except presenters?

Reply:
@SergeiMikhailovskii You may inject anything you want. Ideally all the classes can be injectable and provided as a dependency into the other classes. It could be presenters' dependencies, dependencies of the presenters' dependencies, activities' dependencies, fragments' dependencies and so forth.